### DISCUSSION TITLE
Install error

### DISCUSSION THREAD

**Discussion by gratefulEachOne** [dengxiaoning/cesium_dev_kit#43] [Q&A] [answered]
npm install error ? 
![图片](https://github.com/dengxiaoning/cesium_dev_kit/assets/168287691/a12399bb-0dcc-40a6-9221-0e447a6ae99a)

**Comment by dengxiaoning** [comment, marked as answer]
element-plus@1.0.2-beta.41 conflicts with vue@3.2.13. You need to use parameter force to forcibly install the dependency,
 for example:`npm i --force`
